fe/fe-core/src/main/java/com/starrocks/catalog/CatalogRecycleBin.java

```diff
@@ -279,6 +279,38 @@ private synchronized boolean canErase(long id, long currentTimeMs) {
         return latencyMs > expireMs;
     }
 
+
+    private synchronized boolean canEraseTable(RecycleTableInfo tableInfo, long currentTimeMs) {
+        if (timeExpired(tableInfo.getTable().getId(), currentTimeMs)) {
+            return true;
+        }
+
+        // database is force dropped, the table can not be recovered, erase it.
+        if (GlobalStateMgr.getCurrentState().getDbIncludeRecycleBin(tableInfo.getDbId()) == null) {
```

Comment:
If the table is dropped forced, why does it exist in the recycle bin?

Reply:
If database is dropped force, table that dropped before can exist in the recycle bin.